pkg/broker/filter/filter_handler.go

```diff
@@ -222,7 +222,7 @@ func (r *Handler) sendEvent(ctx context.Context, tctx cloudevents.HTTPTransportC
 	if err != nil {
 		r.logger.Error("Unable to parse subscriberURI", zap.Error(err), zap.String("subscriberURIString", subscriberURIString))
 		// Record the event count.
-		r.reporter.ReportEventCount(reportArgs, err)
+		r.reporter.ReportEventCount(reportArgs, http.StatusNotFound)
```

Comment:
This might be better as a `500`. The config is super broken at this point.

Reply:
Done, thanks!